### PR TITLE
Improve Manager Log by adding a filter for class key and item

### PR DESCRIPTION
**Pull request by matdave**
### What does it do?

Adds an option to search by Class Key or Item ID to the manager logs
image
### Why is it needed?

It's incredibly hard to quickly see the history of who updated a specific element 
### Related issue(s)/PR(s)

Issue #13004

### CODE REVIEW

**Comment by sottwell** [comment]
Looks good to me. +1

**Comment by pixelchutes** [comment]
👍 Nice work! This is awesome.

**Comment by OptimusCrime** [comment]
:+1:

**Comment by amdbuilder** [comment]
:+1:

**Comment by ilyautkin** [comment]
:+1:

**Comment by Mark-H** [comment]
Hey folks, there's a new feature on GitHub where you can 👍 on the first message (or any other message). If you don't mind, using that instead of posting separate messages for each will help keep things less cluttered, especially when there's a feature or fix that needs more discussion or back-and-forth before it is merged ;) Thanks!

File: core/model/modx/processors/system/log/getlist.class.php [hunk 1]
```diff
@@ -69,6 +71,8 @@ public function getData() {
         /* check filters */
         $wa = array();
         if (!empty($actionType)) { $wa['action:LIKE'] = '%'.$actionType.'%'; }
+        if (!empty($classKey)) { $wa['classKey:LIKE'] = '%'.$classKey.'%'; }
```
Comment (Mark-H) [inline core/model/modx/processors/system/log/getlist.class.php]: Does this have to be a LIKE? Are there situations where the class key isn't exactly what was chosen?

Comment (Jako) [inline core/model/modx/processors/system/log/getlist.class.php]: Maybe to search snippet instead of modSnippet

Comment (matdave) [inline core/model/modx/processors/system/log/getlist.class.php]: What Jako said, also just keeping with the same search standards of the other fields

**Comment by OptimusCrime** [comment]
Any reason not to merge this? Should the `LIKE` be rewritten as equals instead? All in favor of merging this into the next minor release.

**Comment by matdave** [comment]
The reason i really prefer `LIKE` for both class key and action is if you want to check all "save" actions across the board or just shorthand Chunk in search

**Comment by OptimusCrime** [comment]
I'm in favor of this too

**Comment by matdave** [comment]
*bump*

**Pull request merged**